includes/MediaWiki/Request/EditRequestParser.php

```diff
@@ -118,7 +112,8 @@ public function parseRequestInterface( RequestInterface $request ): EditRequest
 		return new EditRequest(
 			$reconcilePropertyId,
 			$inputEntity,
-			$otherItems
+			array_key_exists( 'token', $requestBody ) ? $requestBody['token'] : '',
```

Comment:
This shouldn’t be necessary, I think, since we already specify a default value for the `JsonBodyValidator`.